### PR TITLE
SR 1.1.1 Add support for showing daily standard deviation on inflammation data

### PR DESCRIPTION
**Pull request by cptboris**
Extend the graphs to show the daily standard deviation as required by SR 1.1.1

### CODE REVIEW

File: inflammation/models.py [hunk 1]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
```
Comment (hestervennema) [inline inflammation/models.py:38]: not so clear variable name

File: inflammation/models.py [hunk 2]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
...
+    devs = []
+    for entry in data:
+        devs.append((entry - mmm) * (entry - mmm))
+
+    s_dev2 = sum(devs) / len(data)
```
Comment (hestervennema) [inline inflammation/models.py:43]: variable and function name very similar, could get confusing

File: inflammation/models.py [hunk 3]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
...
+    return {'standard deviation': s_dev2}
```
Comment (hestervennema) [inline inflammation/models.py:44]: Im not familiar with this syntax, but is it a pure funtion now? Or is the output a string?

File: tests/test_models.py [hunk 4]
```diff
@@ -29,3 +31,13 @@ def test_daily_mean_integers():
...
 
+
+@pytest.mark.parametrize('data, expected_standard_deviation', [
+    ([0, 0, 0], 0.0),
+    ([1.0, 1.0, 1.0], 0),
+    ([0.0, 2.0], 1.0)
+])
+def test_daily_standard_deviation(data, expected_standard_deviation):
+    from inflammation.models import s_dev
+    result_data = s_dev(data)['standard deviation']
```
Comment (hestervennema) [inline tests/test_models.py:42]: Follow up on the purity of s_dev function?

File: inflammation/models.py (hunk 2 above)
Comment (cptboris) [reply inflammation/models.py]: Fixed this in commit b0c7a3c

File: inflammation/models.py (hunk 1 above)
Comment (cptboris) [reply inflammation/models.py]: Fixed this in commit b0c7a3c

File: inflammation/models.py (hunk 3 above)
Comment (cptboris) [reply inflammation/models.py]: It was outputting a dictionary, not sure why? 
Anyway, it is fixed now in commit b0c7a3c by outputting the correct numpy function for standard deviation

File: tests/test_models.py (hunk 4 above)
Comment (cptboris) [reply tests/test_models.py]: Fixed this in commit b0c7a3c

**Review by hestervennema — COMMENTED** [review]
Looks great!

**Review by hestervennema — APPROVED** [review]

**Pull request merged**